vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -142,11 +142,12 @@ void ApplyArguments(ReactNative::ReactNativeHost const &host, std::wstring const
           hostImpl->InstanceSettings().DebuggerBreakOnNextLine(true);
           hostImpl->InstanceSettings().DebuggerPort(port);
         }
+      } else if (token == L"--proxy") {
+        hostImpl->InstanceSettings().UseWebDebugger(true);
+        hostImpl->InstanceSettings().UseDirectDebugger(false);
+        hostImpl->InstanceSettings().DebuggerBreakOnNextLine(true);
```

Comment:
My understanding is that break on next line works only if direct debugging is enabled, so this line could probably be omitted.